Polygon_repair_2/doc/Polygon_repair_2/Concepts/MultiPolygonWithHoles_2.h

```diff
@@ -8,7 +8,7 @@
  * \cgalHasModel `CGAL::Multipolygon_with_holes_2<Polygon>`
  */
 
-class MultiPolygonWithHoles_2 {
+class MultipolygonWithHoles_2 {
```

Comment:
It would be good to see the formal definition. I guess they describe the correct topology but have geometric defects. Do you also describe the topological nesting.  That is a country with a lake, an island on the lake and a pond in gthe middle of the island.